### DISCUSSION TITLE
feature: on-hit sprite animation

### DISCUSSION THREAD

**Discussion by Uyarrr** [AscensionGameDev/Intersect-Engine#2215] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently we cannot render frames when players/mobs take hit. I think it will be really useful and it's currently missing.

**Comment by jessidion** [comment]
At least a sound/animation would be great :)

And if the blow is a killing blow, it could not play, so a death sound can play instead.

**Comment by Arufonsu** [comment]
> And if the blow is a killing blow, it could not play, so a death sound can play instead.

In the scenario of a finishing hit, both: attack sound/animation and death sound/animation should be played at the same time, thus, both mechanics shouldn't be tied together, so this feat request should be for on-hit sounds/animations only for now.
